BEST_PRACTICES.md

```diff
@@ -123,8 +136,8 @@ Image names should be named consistently to preserve organization and developer
 
 **For example:**
 
-* `refresh-bar-button-item` / `refresh-bar-button-item@2x` and `refresh-bar-button-item-selected` / `refresh-bar-button-item-selected@2x`
```

Comment:
depends on what is being drawn/if it's being cached etc. If it's for instance a border, it should probably be done in code, instead of using an asset. I think same goes for tinting. Instead of having differently tinted assets, just a template asset and tinting in code. Reducing the number of assets to me is a good thing. 👍 

But when drawing gets expensive, or we can't draw it easily, then probably use assets. I'm generally for drawing up until the point I have to override `drawInRect:` or something similar.